ui/components/app/collectibles-tab/collectibles-tab.js

```diff
@@ -38,31 +41,52 @@ export default function CollectiblesTab({ onAddNFT }) {
   const t = useI18nContext();
   const dispatch = useDispatch();
 
-  const collections = {};
-  collectibles.forEach((collectible) => {
-    if (collections[collectible.address]) {
-      collections[collectible.address].collectibles.push(collectible);
-    } else {
-      const collectionContract = collectibleContracts.find(
-        ({ address }) => address === collectible.address,
-      );
-      collections[collectible.address] = {
-        collectionName: collectionContract?.name || collectible.name,
-        collectionImage:
-          collectionContract?.logo || collectible.collectionImage,
-        collectibles: [collectible],
-      };
-    }
-  });
+  const getCollections = () => {
+    const collections = {};
+    const previouslyOwnedCollection = {
+      collectionName: 'Previously Owned',
+      collectibles: [],
+    };
+    collectibles.forEach((collectible) => {
+      if (collectible?.isCurrentlyOwned === false) {
+        previouslyOwnedCollection.collectibles.push(collectible);
+      } else if (collections[collectible.address]) {
+        collections[collectible.address].collectibles.push(collectible);
+      } else {
+        const collectionContract = collectibleContracts.find(
+          ({ address }) => address === collectible.address,
+        );
+        collections[collectible.address] = {
+          collectionName: collectionContract?.name || collectible.name,
+          collectionImage:
+            collectionContract?.logo || collectible.collectionImage,
+          collectibles: [collectible],
+        };
+      }
+    });
+    return [collections, previouslyOwnedCollection];
+  };
+
+  const [collections, previouslyOwnedCollection] = getCollections();
 
   const onEnableAutoDetect = () => {
     history.push(EXPERIMENTAL_ROUTE);
   };
 
+  const onRefresh = () => {
+    if (isMainnet) {
+      dispatch(detectCollectibles());
+    }
+    dispatch(checkAndUpdateCollectiblesOwnershipStatus());
```

Comment:
Here is the change to accompany [this suggestion](checkAndUpdateCollectiblesOwnershipStatus):
```suggestion
    checkAndUpdateCollectiblesOwnershipStatus();
```

We're not calling `await` here, so the result of this call is unhandled. But that seems OK in this case where we don't expect failure, since if it rejects it will still be captured by Sentry.

Reply:
updated here: https://github.com/MetaMask/metamask-extension/pull/13110/commits/cf5fa0ce01ba5ef3be07cc4fcd8617f27874e141